src/nvapi_gpu.cpp

```diff
@@ -95,6 +95,34 @@ extern "C" {
         return Ok(n);
     }
 
+    NvAPI_Status __cdecl NvAPI_GPU_GetBusSlotId(NvPhysicalGpuHandle hPhysicalGpu, NvU32 *pBusSlotId) {
+        constexpr auto n = __func__;
+
+        if (nvapiAdapterRegistry == nullptr)
+            return ApiNotInitialized(n);
+
+        if (hPhysicalGpu == nullptr)
+            return InvalidArgument(n);
+
+        *pBusSlotId = 0; // just assume zero...
+
+        return Ok(n);
+    }
+
```

Comment:
Maybe you could use nvml: nvmlDeviceGetPciInfo_v3(nvmlDevice_t device, nvmlPciInfo_t *pci); and you should get the id from: pci.device ?

Reply:
From my understanding, slot ID and bus ID are unfortunately not the same.

Reply:
Ok. It would have been too easy for nVidia to not use ambiguity in their header language i guess.. :roll_eyes:

Reply:
Nah, please refrain bashing NVIDIAs choices in nvapi, at least here in this project :)
Nvapi has a history of I guess 25 years with probably lots of different developers and lots of different functionalities while still being backwards compatible all the way back. I can tell you from own experiences that it is impossible to do this without having places where history shows, especially with something evolving that fast as GPU architecture and GPU use cases.

Reply:
It is probably not a huge issue, since most usage of dxvk-nvapi would be from a GPU connected to pci-e anyway... I mean, would anyone realistically use anything less than a GTX2xx series card now? And if so - what would the REAL problems be with reporting an AGP card as pcie?

Reply:
Yeah, let’s wait first if something of this PR is needed at all, considering the first real issue is/was the adapter LUID in cuda. If possible, I would prefer not to add implementations for those two GPU_ methods. The other two methods are probably ok to add, if those make difference.